RaveBusinessLogic/BRAT.xml

```diff
@@ -24,10 +19,6 @@
           </Node>
           <Node label="Management">
             <Children collapsed="true">
-              <Node label="Unsuitable or Limited Opportunities" xpath="Vector" type="line" symbology="oPBRC_UD" id="limited" />
-              <Node label="Risk of Undesirable Dams" xpath="Vector" type="line" symbology="oPBRC_UI" id ="risk" />
-              <Node label="Restoration or Conservation Opportunities" xpath="Vector" type="line" symbology="oPBRC_CR" id="opportunity" />
-
               <Node label="Unsuitable or Limited Opportunities" xpath="Geopackage/Layers/Vector[@id='BRAT_RESULTS']" type="line" symbology="limitation" id="limited" />
               <Node label="Risk of Undesirable Dams" xpath="Geopackage/Layers/Vector[@id='BRAT_RESULTS']" type="line" symbology="risk" id="risk" />

```

Comment:
This is the garbage line. The symbology you refer to is: `symbology="risk"` and that is not what we want we want `symbology="oPBRC_UI"` from the original pyBRAT.xml BL:
https://github.com/Riverscapes/RiverscapesXML/blob/691108896b92158cea56ae9f67cfbdb3112f9922/RaveBusinessLogic/pyBRAT.xml#L23